src/Aspire.Hosting/Aspire.Hosting.csproj

```diff
@@ -47,6 +47,7 @@
     <PackageReference Include="KubernetesClient" />
     <PackageReference Include="Microsoft.Extensions.Hosting" />
     <PackageReference Include="Polly.Core" />
+    <PackageReference Include="System.Text.Json" />
```

Comment:
?

Reply:
@davidfowl In VS2022 preview it looks like NuGetAudit is enabled by default. Locally I had to update STJ to latest version.
I updated the PR dropping this PackageReference and keeping a `<NuGetAudit>false</NuGetAudit>` MSBuild property in my local Directory.Build.props.